### PR TITLE
doc: build status markdown format tweaks

### PR DESCRIPTION
**Pull request by detailyang**
now the build status is broken as the following:
screen shot 2017-05-26 at 18 10 04

Also "Document"
screen shot 2017-05-26 at 18 15 45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,8 +53,7 @@ Readable? Efficient? Can your Python/Ruby/V8 do better?
 Status
 ------
 
-**Lua Fun** is in an early alpha stage. The library fully [documented]
-[Documentation] and covered with unit tests.
+**Lua Fun** is in an early alpha stage. The library fully [Documentation] and covered with unit tests.
```
Comment (Totktonada) [inline README.md:56]: `fully [Documentation]` → `fully [documented][Documentation]`

IOW, the fix is to get rid from improper newline.

File: README.md [hunk 2]
```diff
@@ -69,7 +67,7 @@ notice. Please use **stable** branch for your production deployments.
 If you still want to use **master**, please don't forget to grep `git log`
 for *Incompatible API changes* message. Thanks!
 
-Please check out [documentation][Documentation] for more information.
+Please check out [Documentation] for more information.
```
Comment (Totktonada) [inline README.md:70]: This change does not look necessary.

**Comment by detailyang** [comment]
ping @Totktonada .. minor fix (this pr is created at 2017 😂)

**Comment by Totktonada** [comment]
Sorry for bothering you. I'm going to fix this and several other problems in PR #62.

**Comment by Totktonada** [comment]
Fixed in PR #62.

**Pull request closed without merging**